cupy/cuda/stream.py

```diff
@@ -22,7 +22,7 @@ class Event(object):
 
     """
     def __init__(self, block=False, disable_timing=False, interprocess=False):
-        self.ptr = None
+        self.ptr = 0
```

Comment:
It is indeed required for the case that __init__ raises an exception before setting the ptr. If we omit the first line, __del__ tries to access ptr, and raises an AttributeError. This is problematic.